apps/server/src/tasks/dto/createFloatTask.input.ts

```diff
@@ -1,21 +1,28 @@
-import { Field, InputType } from '@nestjs/graphql';
 import { Duration } from 'moment';
+import { Field, InputType } from '@nestjs/graphql';
 import { Interval } from '../../utils/interval.scalar';
 import { ChunkInfoInput } from './chunkInfo.input';
 import { CreateTaskInput } from './createTask.input';
 
+
 // TODO Add some custom validators
 @InputType()
 export class CreateFloatTaskInput extends CreateTaskInput {
   @Field(() => ChunkInfoInput)
   chunkInfo: ChunkInfoInput;
```

Comment:
since minTimeBetweenChunks, min and maxChunkDuration properties are moved here - should we remove this field?